File: gluecodium/src/test/resources/smoke/field_constructors/output/android/com/example/smoke/ImmutableStructNoClash.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class ImmutableStructNoClash {
    @NonNull
    public final String stringField;
    public final int intField;
    public final boolean boolField;
    public ImmutableStructNoClash(@NonNull final String stringField, final int intField, final boolean boolField) {
        this.stringField = stringField;
        this.intField = intField;
        this.boolField = boolField;
    }
    public ImmutableStructNoClash() {
        this.stringField = "nonsense";
        this.intField = 42;
        this.boolField = true;
    }
}

File: gluecodium/src/test/resources/smoke/field_constructors/output/android-kotlin/com/example/smoke/ImmutableStructNoClash.kt
/*

 *
 */

@file:JvmName("ImmutableStructNoClashExtensions")

package com.example.smoke


class ImmutableStructNoClash {
    @JvmField val stringField: String
    @JvmField val intField: Int
    @JvmField val boolField: Boolean



    constructor(stringField: String, intField: Int, boolField: Boolean) {
        this.stringField = stringField
        this.intField = intField
        this.boolField = boolField
    }


    constructor() {
        this.stringField = "nonsense"
        this.intField = 42
        this.boolField = true
    }





}

